### PR TITLE
Webhook eventbatching

### PR DESCRIPTION
**Pull request by anup-git-hub**

### CODE REVIEW

File: VWOSdk.DemoApp/Controllers/HomeController.cs [hunk 1]
```diff
@@ -117,7 +145,18 @@ public IActionResult FeatureTest([FromQuery] string user)
         [HttpGet]
         public IActionResult Push([FromQuery] string user)
         {
-            var userId = string.IsNullOrEmpty(user) ? GetRandomName() : user;
+            var userId = "Tierra"; //string.IsNullOrEmpty(user) ? GetRandomName() : user;
```
Comment (sanyamjain65) [inline VWOSdk.DemoApp/Controllers/HomeController.cs:148]: why hard-coded userId?

File: VWOSdk.DemoApp/Controllers/HomeController.cs [hunk 2]
```diff
@@ -128,5 +167,25 @@ public IActionResult Push([FromQuery] string user)
             var json = new ViewModel(SettingsFile, userId, activateResponse, tagKey, tagValue);
             return View(json);
         }
+        [HttpGet]
+        public IActionResult Flash([FromQuery] string user)
```
Comment (sanyamjain65) [inline VWOSdk.DemoApp/Controllers/HomeController.cs:171]: rename it to "flush"

also flushEvents api should be exposed on vwoClient and it should work in sync mode i.e first the queue data should be sent to VWO servers and then the response of a network call should be returned to user as true or false

File: VWOSdk.DemoApp/Controllers/SettingsProvider.cs [hunk 3]
```diff
@@ -1,53 +1,109 @@
-﻿using System.IO;
+﻿using Microsoft.Extensions.Hosting;
+using System.IO;
 using System.Reflection;
+using VWOSdk;
+using System.Threading.Tasks;
+using System.Collections.Concurrent;
```
Comment (sanyamjain65) [inline VWOSdk.DemoApp/Controllers/SettingsProvider.cs:6]: reason for changes in this file?

File: VWOSdk.DemoApp/Controllers/SettingsProvider.cs [hunk 4]
```diff
@@ -1,53 +1,109 @@
...
-                    }
-                }
+                File.Create(_SettingsFilePath).Close();
+               
             }
-            catch { }
 
-            return null;
+           
+                Settings SettingsFile = VWO.GetSettingsFile(accountId, sdkKey);
```
Comment (sanyamjain65) [inline VWOSdk.DemoApp/Controllers/SettingsProvider.cs:50]: for webhook "getAndUpdateSettings" api should be exposed.

File: VWOSdk.DemoApp/Controllers/SettingsProvider.cs (hunk 3 above)
Comment (anup-git-hub) [reply VWOSdk.DemoApp/Controllers/SettingsProvider.cs]: for testing and get vwo app data

**Pull request merged**